src/routes/(app)/product/[id]/+page.svelte

```diff
@@ -479,7 +488,14 @@
 							{:else}
 								<button
 									formaction="?/addToCart"
-									disabled={!acceptRestriction || loading}
+									disabled={!acceptRestriction ||
+										(!!data.physicalCartMinAmount &&
+											data.product.price.amount * quantity <=
+												toCurrency(
+													data.product.price.currency,
+													data.physicalCartMinAmount,
+													data.currencies.main
+												))}
```

Comment:
you removed the ` || loading` here

Also I think we're still able to add to cart? So it shouldn't be disabled here?